index.html

```diff
@@ -215,26 +215,68 @@ <h3 class="center">Contributors</h3>
       <p class="center">These are the people who have contributed actively more than once with content for our meetings. Without these creators, there would be no FCCVienna.</p>
       <div class="col s12 m4 offset-m2">
         <ul class="normalize-ul center">
-          <li><a href="https://matthiasliszt.github.io">Matthias Liszt</a></li>
-          <li><a href="https://github.com/42pre">Martin Plochberger</a></li>
-          <li><a href="https://github.com/edorivai">Edo Rivai</a></li>
-          <li><a href="https://github.com/bonny3d">Bonny Dongre</a></li>
-          <li><a href="https://github.com/abisz">Simon Reinsperger</a></li>
-          <li>Markus Hösel</li>
+          <li>
+            <img class="profile-picture" src="https://github.com/matthiasliszt.png" />
+            <a href="https://matthiasliszt.github.io">Matthias Liszt</a>
+          </li>
+          <li>
+            <img class="profile-picture" src="https://github.com/42pre.png" />
+            <a href="https://github.com/42pre">Martin Plochberger</a>
+          </li>
+          <li>
+            <img class="profile-picture" src="https://github.com/edorivai.png" />
+            <a href="https://github.com/edorivai">Edo Rivai</a>
+          </li>
+          <li>
+            <img class="profile-picture" src="https://github.com/bonny3d.png" />
+            <a href="https://github.com/bonny3d">Bonny Dongre</a>
+          </li>
+          <li>
+            <img class="profile-picture" src="https://github.com/abisz.png" />
+            <a href="https://github.com/abisz">Simon Reinsperger</a>
+          </li>
+          <li>
+            <img class="profile-picture" src="https://github-images.s3.amazonaws.com/enterprise/2.8/assets/images/help/profile/identicon.png" />
+            Markus Hösel
+          </li>
+          <li>
+            <img class="profile-picture" src="https://github.com/achim-tecklenburg.png" />
+            <a href="https://github.com/achim-tecklenburg">Achim Tecklenburg</a>
+          </li>
```

Comment:
Could you also add Monika, @calmandniceperson? She is now [in the list](https://github.com/FCCVienna/FCCVienna.github.io/issues/36#issuecomment-334709076). I forgot to add her to the list, and totally my mistake. If you'd rather not, I can add her once this is in master. Will approve for now.

Reply:
Done. :)